release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -768,6 +768,37 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+
+* Previously, when you created an application using source code from Git repositories with the `.git` suffix, and then clicked the edit source code link, a "page not found" error was displayed. This fix removes the `.git` suffix from the repository URL and transforms the SSH URL to an HTTPS URL. The generated link now leads to the correct repository page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896296[*BZ#1896296*])
+
+* Previously, the underlying `SinkBinding` resources were shown in the *Topology* view, along with the actual source created in the case of `Container`  and `KameletBinding` resources, confusing users. This issue was fixed. Now, only the actual resource created for the event source is displayed in the *Topology* view, and the underlying `SinkBinding` resources, if created, are displayed in the sidebar. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906685[*BZ#1906685*])
+
+* Previously, when you installed the Serverless Operator, without creating the eventing custom resource, a channel card was displayed. When you clicked the card, a confusing alert message was displayed. This issue has now been fixed. The channel card, with a proper alert message, is now displayed only if the channel custom resource definition is present. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909092[*BZ#1909092*])
+
+* Previously, when you closed the web terminal connection, all the terminal output from that session, which can be useful to you, disappeared. This issue has been fixed. The terminal output is now retained even after the session is closed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909067[*BZ#1909067*])
+
+* Technology preview badges were displayed on the *Eventing* user interface although it had its GA release. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894810[*BZ#1894810*])
+
+* Previously, volume mounts for deployments were not preserved if the deployment was edited using the console edit flows. The modified deployment YAML overwrote or removed the volume mounts in the pod template specification. This issue has been fixed. The *volume mounts* are now preserved even when the deployment is edited using the console edit flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1867965[*BZ#1867965*])
+
+* If triggers used a Knative service and the In Memory Channel as subscriber, the Knative services and resources were not displayed on the *Topology* view. This issue has been fixed now, so that the Knative data model returns proper data, and the Knative resources are displayed on the *Topology* view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906683[*BZ#1906683*])
```

Comment:
```suggestion
* In case of multiple triggers one subscribing to Knative service and another to In Memory Channel as subscriber, the Knative resources were not displayed on the *Topology* view. This issue has been fixed now, so that the Knative data model returns proper data, and the Knative resources are displayed on the *Topology* view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906683[*BZ#1906683*])
```

Reply:
Done!